tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml

```diff
@@ -6,7 +6,6 @@ tests:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
-      - nrf54h20dk/nrf54h20/cpuapp
```

Comment:
So from now we cannot use Oberon crypto APIs directly on nRF54H20? If so, then maybe we should also add dependency that prevents enabling `CONFIG_BT_FAST_PAIR_CRYPTO_OBERON` on nRF54H20. Looking at the rest of your PR, we might also need to update the dependencies of the `CONFIG_BT_FAST_PAIR_CRYPTO_PSA`: `MBEDTLS_PSA_CRYPTO_C` should not be selected, right?

Ref: https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/services/fast_pair/fp_crypto/Kconfig.fp_crypto#L21